h2/src/main/org/h2/command/Parser.java

```diff
@@ -2198,17 +2198,24 @@ private Prepared parseComment() {
             // if the db name is equal to the schema name
             ArrayList<String> list = Utils.newSmallArrayList();
             do {
-                list.add(readUniqueIdentifier());
+                if (currentTokenType == DOT) {
```

Comment:
comment on catalog..tablename, if the defaultschemaname geht left out

Reply:
https://github.com/h2database/h2database/pull/2092#discussion_r319490143

Reply:
I searched how often this is used in our code, and it is. Since it is officially allowed in sybase and I think in mssql as well, I would like to keep it supported. I also tried this in postgres, there it is not possible.